### DISCUSSION TITLE
What does Nuclei do when it starts?

### DISCUSSION THREAD

**Discussion by stefanocoding** [projectdiscovery/nuclei#4111] [Q&A] [answered]
Hi community.
If I run Nuclei with a IPv6 subnet like `nuclei -u "2620:134:b000::/40"`, I only see the Nuclei banner indefinitely. No idea what's going on. 
What is it doing when it starts?
Thanks in advance for the answers.

**Comment by ehsandeep** [comment, marked as answer]
@stefanocoding, the tool is attempting to process the input. specifically, it fails when trying to expand the IPv6 range `2620:134:b000::/40` using [mapcidr](https://github.com/projectdiscovery/mapcidr), causing it to hang. Are you actually intending to scan this range?

**Comment by stefanocoding** [comment]
@ehsandeep thank you for the explanation! No, I don't intend to scan that range. I noticed the issue when testing a template with a list of random targets from bug bounties programs which had an IPv6 subnet there. I removed it and everything worked.
Is it possible to get a warning if it fails or something like that? Because I tried the verbose flag but it didn't provide more information.

**Reply by ehsandeep** [reply]
Thanks and yes it's an edge case and error handling can be improved.

**Reply by stefanocoding** [reply]
🙂 👍
